panther_analysis_tool/main.py

```diff
@@ -669,11 +669,15 @@ def test_analysis(args: argparse.Namespace) -> Tuple[int, list]:
         return 1, ["Nothing to test in {}".format(args.path)]
 
     # Apply the filters as needed
+    if getattr(args, "filter_inverted", None) is None:
+        args.filter_inverted = {}
```

Comment:
Why do you do this input normalization here and not in the `zip_analysis` function when you use this argument?

If it's because it only needs to happen once and test always happens before zip, I think we should we move this setup to the `def run()` function on line ~1500.

Reply:
The reason I did not put this in the `run` function is because the unit tests directly invoke `zip_analysis`

I'll move it into `zip_analysis` to avoid adding extra complexity to the tests

Reply:
Moved this down into `run` and adjusted the failing unit tests to manually set the `args.filter_inverted`